cypress.config.js

```diff
@@ -1,41 +1,30 @@
 const { defineConfig } = require('cypress');
-const faker = require('@faker-js/faker');
-const { clear } = require('./server/db');
-const { seed } = require('./server/db');
+const { faker } = require('@faker-js/faker');
 const {
   addMatchImageSnapshotPlugin
 } = require('cypress-image-snapshot/plugin');
 
 module.exports = defineConfig({
   e2e: {
-    baseUrl: 'http://localhost:1667/',
+    baseUrl: 'https://conduit.mate.academy/',
     setupNodeEvents(on, config) {
       on('task', {
         generateUser() {
           const randomNumber = Math.ceil(Math.random(1000) * 1000);
```

Comment:
The expression `Math.ceil(Math.random(1000) * 1000)` is intended to generate a random number between 1 and 1000. However, `Math.random()` generates a number between 0 (inclusive) and 1 (exclusive), and the parameter `1000` inside `Math.random()` is unnecessary. Consider using `Math.ceil(Math.random() * 1000)` to achieve the desired range.